python/ray/ml/train/integrations/horovod/horovod_trainer.py

```diff
@@ -0,0 +1,239 @@
+from typing import Dict, Callable, Optional, Union
+
+from ray.ml.config import ScalingConfig, RunConfig
+from ray.ml.trainer import GenDataset
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.checkpoint import Checkpoint
+
+
+from ray.ml.train.data_parallel_trainer import DataParallelTrainer
+from ray.train.horovod import HorovodConfig
+
+
+class HorovodTrainer(DataParallelTrainer):
+    """A Trainer for data parallel Horovod training.
+
+    This Trainer runs the function ``train_loop_per_worker`` on multiple Ray
+    Actors. These actors already have the necessary Horovod process group already
+    configured for distributed Horovod training.
```

Comment:
I don't think process group is a concept for Horovod training.

```suggestion
    Actors. These actors already have the necessary Horovod setup already configured for distributed Horovod training.
```